main.nginx.conf

```diff
@@ -95,12 +97,15 @@ http {
     }
 
     location / {
-      root ./dist;
-
-      location /index.html {
-        include ./common-headers.nginx.conf;
-        add_header Cache-Control no-cache;
-      }
+      proxy_pass http://localhost:5000/;
```

Comment:
I had to change this to 5002 to run the app

Reply:
Do you have other services running on ports 5000 & 5001?  I'll add a note that this change will not be included in final PR.